frontend/src/App.tsx

```diff
@@ -15,6 +17,8 @@ function App() {
             <Route index element={<HomePage />} />
             <Route path="add-blog" element={<AddBlog />} />
             <Route path="details-page/:title/:postId" element={<DetailsPage />} />
+            <Route path="user-auth" element={<SignIn />} />
```

Comment:
Lets make it signin, so that we are consistent to signup

Reply:
Okay.